src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -414,6 +416,12 @@ blockCommands = M.fromList $
   , "pagebreak"
   ]
 
+blockTextcolor :: PandocMonad m => LP m Blocks
+blockTextcolor = do
+    skipopts 
+    color <- braced
+    divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
+
```

Comment:
Small problem here:  you should add a `try` before the initial `do`.  Otherwise, you'll get a failure if the `\color` command is parsed and the `notFollowedBy` fails, because the whole parser will have failed after having consumed some text.